packages/react-admin/src/reducer/admin/resource/list/selectedIds.js

```diff
@@ -0,0 +1,26 @@
+import {
+    SET_LIST_SELECTED_IDS,
+    TOGGLE_LIST_ITEM,
+} from '../../../../actions/listActions';
+
+const initialState = [];
+
+export default (previousState = initialState, action) => {
+    if (action.type === SET_LIST_SELECTED_IDS) {
```

Comment:
Is there a single state for all list selections ?
```
// from listActions.js
export const setListSelectedIds = (resource, ids) => ({
    type: SET_LIST_SELECTED_IDS,
    payload: ids,
    meta: { resource },
});
```

Reply:
I don't get the question

Reply:
Oh you mean a single one for all resources ?

Reply:
yes :)

Reply:
Ok, so no there is a `selectedIds` sub reducer for each resource

Reply:
👍 for `selectedIds` per resource.